recipes/acctaudem/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "acctaudem" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/RyanMcCarthy-NOAA/accTauDEM/archive/{{ version }}.tar.gz
+  sha256: b35bac0ea056991aefe93a4330a8574e171f7d562c7e6e909f95335b01758e6b
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - mpich
+    - make
+    - cmake
+  host:
+    - libgdal
+    - mpich
```

Comment:
Is this mpich-only?  You should look at https://conda-forge.org/docs/maintainer/knowledge_base.html#message-passing-interface-mpi

Reply:
The README says to use mpich. It also says openmpi is an option, but that is it untested. So I figured it say to stick with mpich.